duckiebot/camera/command.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+import os
+import platform
+import socket
+import subprocess
+
+from dt_shell import DTCommandAbs, DTShell, dtslogger
+from dt_shell.env_checks import check_docker_environment
+from utils.cli_utils import start_command_in_subprocess
+from utils.networking_utils import get_duckiebot_ip
+from utils.docker_utils import (
+    bind_duckiebot_data_dir,
+    default_env,
+    get_remote_client,
+    remove_if_running,
+    pull_if_not_exist,
+    check_if_running,
+    get_endpoint_architecture,
+)
+from utils.duckietown_utils import get_distro_version
+from utils.misc_utils import sanitize_hostname
+
+DT_INTERFACE_IMAGE = "duckietown/dt-duckiebot-interface:daffy-{arch}"
+
+
+class DTCommand(DTCommandAbs):
+    @staticmethod
+    def command(shell: DTShell, args):
+
+        prog = "dts duckiebot camera DUCKIEBOT_NAME --stop/--start"
+        usage = """
+This command can prevent the camera node to start in the dt-duckiebot-interface. 
+
+Setup:
+
+    %(prog)s
+"""
+
+        parser = argparse.ArgumentParser(prog=prog, usage=usage)
+        parser.add_argument("hostname", default=None,
+                            help="Name of the Duckiebot")
+
+        parser.add_argument(
+            "--stop", dest="stop", default=None, action="store_true", help="Stop the camera node"
+        )
+
+        parser.add_argument(
+            "--start", dest="start", default=None, action="store_true", help="Start the camera node"
+        )
+
+        parsed = parser.parse_args(args)
+
+        if parsed.stop and parsed.start:
```

Comment:
Should you also check if both `stop` and `start` are false? In this case it would also be bad arguments right?

Reply:
Done!